src/components/Modal.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {FormEvent, useState} from 'react'
+import './Modal.scss'
+import {createPortal} from "react-dom"
+import ModalContent from "./ModalForm"
+
+export default function WelcomingModal() {
```

Comment:
because there can be more than one exported object from this file, it would be best if we could keep from using default exports and just do export. this approach should also force us to used named imports and keep naming consistent throughout the project